### PR TITLE
Stylin

### PR DESCRIPTION
**Pull request by s-steel**
## Description
Implement styling throughout the app using bootstrap

## Motivation and Context
Creates uniform style throughout the app and makes it look nice!

## How Has This Been Tested?
Views have been tested checking for css
All tests passing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have updated the README to reflect any changes.

### CODE REVIEW

**Review by carson-jardine — APPROVED** [review]
Lookin good!!

**Review by ETBassist — APPROVED** [review]
Such style!

File: app/views/footprints/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    </section> 
+  <% end%>
+</div>
+
+<% unless @cars.empty? %>
+  <section class='calculate-footprint'>
+    <div class="form-group container-lg text-center">
+      <%= form_tag '/carbon_calculator', method: :post do %>
+        <div class='form-select form-select-lg mb-3 p-2'>
+          <%= select_tag(:car, options_for_select(@cars.map {|car| "#{car.year} #{car.make} #{car.model}"}, '1')) %>
```
Comment (ETBassist) [inline app/views/footprints/new.html.erb:21]: I actually just noticed this - I think in the backend we're expecting the car id in the mutation to create a new footprint, so we're going to need to get that into this form, which means we'll probably need it in the car facade and poro too (we just need to add it to the query)

**Review by ETBassist — CHANGES_REQUESTED** [review]
Left a comment below; should be a pretty easy fix, so I'm going to crank it out tonight but wanted to make sure it doesn't get merged in before that

**Review by ETBassist — APPROVED** [review]

**Review by c-bartell — APPROVED** [review]

**Pull request merged**